### PR TITLE
Gardener and Healer Fixes

### PR DESCRIPTION
**Pull request by Ultimate-Fluff**
- I had to fix a bit of shading on the Healer drone because my OCD dictated it.
- Fixed a stray pixel floating around on one of Gardener's icon states. Also added the sac to certain missing icon states.
- Changed some flavor text to add a bit more lore.
- Changed Healer's heal amount for each xenomorph from 10 to 12 health a second for an equal exchange of health. Before, she sacrificed 120 health for each 100 damage she heals. If requested, will revert it, **but I really do not want to.**

# Explain why it's good for the game

# Testing Photographs and Procedure
 <details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

 

:cl: Puckaboo2
tweak: Changed some text for Healer and Gardener's abilities for lore purposes.
tweak: Brightened and darkened portions of Healer's markings in certain directions for perspective purposes.
tweak: Added a sac for the Gardener Dead overlay icon state and tweaked the overlay's knocked down icon state.
bugfix: Removed a few floating pixels on the gardener overlay's icon states. Spooky.
balance: Buffed Healer's heal amount from 10 to 12 health a second for an equivalent exchange.
rscadd: Gave Healer the ability to heal xenos from allied hives. She will also not heal xenos from hostile hives.
/:cl:

### CODE REVIEW

**Review by Agameofscones — APPROVED** [review]
sprotes approved

**Comment by Ultimate-Fluff** [comment]
> sprotes approved

Are the balance changes approved?

File: code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/gardener.dm [hunk 1]
```diff
@@ -92,24 +92,24 @@
...
 
 	if(!istype(target_turf))
-		to_chat(xeno, SPAN_WARNING("You can't do that here."))
+		to_chat(xeno, SPAN_WARNING("You cannot plant a fruit without a weed garden."))
 		return
 
 	var/obj/effect/alien/weeds/target_weeds = locate(/obj/effect/alien/weeds) in target_turf
 	if(!target_weeds)
-		to_chat(xeno, SPAN_WARNING("There's no weed to place it on!"))
+		to_chat(xeno, SPAN_WARNING("The are no weeds to plant a fruit within!"))
```
Comment (harryob) [inline code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/gardener.dm:100]: there

File: code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/gardener.dm [hunk 2]
```diff
@@ -92,24 +92,24 @@
...
 		return
 
 	if(locate(/obj/effect/alien/resin/trap) in range(1, target_turf))
-		to_chat(xeno, SPAN_XENOWARNING("This is too close to a resin hole!"))
+		to_chat(xeno, SPAN_XENOWARNING("This location is too close to a resin hole!"))
 		return
 
 	if(locate(/obj/effect/alien/resin/fruit) in range(1, target_turf))
-		to_chat(xeno, SPAN_XENOWARNING("This is too close to another fruit!"))
+		to_chat(xeno, SPAN_XENOWARNING("This location too close to another fruit!"))
```
Comment (harryob) [inline code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/gardener.dm:112]: location is

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/gardener.dm (hunk 2 above)
Comment (Ultimate-Fluff) [reply code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/gardener.dm:112]: > location is

I fixed it before I had to do a soft reset to squash everything so it was reverted. Sorry!

**Review by harryob — APPROVED** [review]

File: code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/gardener.dm (hunk 2 above)
Comment (harryob) [reply code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/gardener.dm]: no prob

**Pull request merged**